nucypher/cli/commands/bob.py

```diff
@@ -165,14 +179,12 @@ def bob():
 @option_config_root
 @group_general_config
 def init(general_config, config_options, config_root):
-    """
-    Create a brand new persistent Bob.
-    """
-    emitter = _setup_emitter(general_config)
+    """Create a brand new persistent Bob."""
+    emitter = setup_emitter(general_config)
```

Comment:
banner?

Reply:
Wow good eye - Yeah this is something that I'd like to follow up on after this PR.

Banners are too noisy anyways, no?  :-)